packages/js-sdk/tests/sandbox/files/write.test.ts

```diff
@@ -1,12 +1,120 @@
-import { assert } from 'vitest'
+import path from 'path'
+import { assert, onTestFinished } from 'vitest'
 
+import { WriteEntry } from '../../../src/sandbox/filesystem/index.js'
 import { sandboxTest } from '../../setup.js'
 
+sandboxTest('write file', async ({ sandbox }) => {
+  const filename = 'test_write.txt'
+  const content = 'This is a test file.'
+
+  // Attempt to write with undefined path and content
+  await sandbox.files
+    // @ts-ignore
+    .write(undefined, content)
+    .then((e) => {
+      assert.isUndefined(e)
+    })
+    .catch((err) => {
+      assert.instanceOf(err, Error)
+      assert.include(err.message, 'Cannot read properties of undefined')
+    })
+
+  const info = await sandbox.files.write(filename, content)
+  assert.isFalse(Array.isArray(info))
+  assert.equal(info.name, filename)
+  assert.equal(info.type, 'file')
+  assert.equal(info.path, `/home/user/${filename}`)
+
+  const exists = await sandbox.files.exists(filename)
+  assert.isTrue(exists)
+  const readContent = await sandbox.files.read(filename)
+  assert.equal(readContent, content)
+})
+
+sandboxTest('write multiple files', async ({ sandbox }) => {
+  // Attempt to write with empty files array
+  const emptyInfo = await sandbox.files.write([])
+  assert.isTrue(Array.isArray(emptyInfo))
+  assert.equal(emptyInfo.length, 0)
+
+  // Attempt to write with undefined path and file array
+  await sandbox.files
+    // @ts-ignore
+    .write(undefined, [{ path: 'one_test_file.txt', data: 'This is a test file.' }])
+    .then((e) => {
+      assert.isUndefined(e)
+    })
+    .catch((err) => {
+      assert.instanceOf(err, Error)
+      assert.include(err.message, 'Cannot read properties of undefined')
```

Comment:
Can we explicitly mention in the error message that a user forgot to specify a path?